cmd/aries-agent-mobile/pkg/wrappers/config/options.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package config
+
+// Options represents configurations for Aries
+type Options struct {
+	UseLocalAgent bool
+
+	URL      string
+	APIToken string
+
+	Label                string
+	AutoAccept           bool
+	TransportReturnRoute string
+	LogLevel             string
+	DBNamespace          string
+
+	// expected to be ignored by gomobile
+	// not intended to be used by golang code
+	HTTPResolvers     []string
+	OutboundTransport []string
+}
+
+// AddHTTPResolver appends an http resolver url to the options
```

Comment:
replace ‘an’ with a http

Reply:
i looked this up online for clarification, it seems the pronunciation could go either way. it looks like `an http` is used more than `a http` in our codebase - 15 for an and 3 for a. as a solution, maybe we should select `an http` and use that all around

Reply:
"an http" sounds correct to me